components/Footer.tsx

```diff
@@ -2,27 +2,34 @@ import React from 'react'
 import links from '../utils/Links.store'
 import Image from 'next/image'
 import Link from 'next/link'
+import { FooterBlockProps } from '../interface'
 
-const Footer = () => {
+const Footer: React.FC = () => {
   return (
-    <footer>
-      <div className='container flex flex-wrap'>
+    links && (
+      <footer>
         <div className='container flex flex-wrap'>
-          {links.map((ele, i) => {
-            return <FooterBlock {...ele} key={i} />
-          })}
+          <div className='container flex flex-wrap'>
+            {links.map((ele, i) => {
+              return <FooterBlock {...ele} key={i} />
+            })}
+          </div>
+          <Link href={links[0].vercel!.href}>
+            <a className='banner' target={links[0].vercel!.target}>
+              <Image
+                className=''
+                src={links[0].vercel!.img}
```

Comment:
From ts docs: https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#non-null-assertion-operator-postfix-

Can we use `links[0].vercel?.target || ""`?